ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/snmp-settings.j2

```diff
@@ -4,16 +4,26 @@
 
 ### SNMP Configuration Summary
 
-| Contact | Location | SNMP Traps |
-| ------- | -------- | ---------- |
+| Contact | Location | SNMP Traps | State |
+| ------- | -------- | ---------- | ----- |
 {%     set row_contact = snmp_server.contact | arista.avd.default('-') %}
 {%     set row_location = snmp_server.location | arista.avd.default('-') %}
-{%     if snmp_server.traps.enable | arista.avd.default(false) %}
-{%         set row_traps = 'Enabled' %}
+{%     if snmp_server.traps.enable is arista.avd.defined(true) %}
+{%         set row_state = 'Enabled' %}
 {%     else %}
-{%         set row_traps = 'Disabled' %}
+{%         set row_state = 'Disabled' %}
+{%     endif %}
+| {{ row_contact }} | {{ row_location }} | All | {{ row_state }} |
+{%     if snmp_server.traps.snmp_traps is arista.avd.defined %}
+{%         set row_traps_disabled = snmp_server.traps.snmp_traps | selectattr('enabled','false') | map(attribute='name') | arista.avd.natural_sort | join('<br>') %}
```

Comment:
please join on `', '`.
Ensure to follow style with space after comma, space around `=`.
Same below

Reply:
Because the snmp_traps are the custom commands, like <snmp_trap sub_trap>. It would be confusing if we join with "," So I used <br> to make sure different commands are in different lines.

Reply:
I will make changes for space around ```=```